README.md

```diff
@@ -1,12 +1,29 @@
 
-[![CircleCI](https://circleci.com/gh/woocommerce/woocommerce-ios.svg?style=svg)](https://circleci.com/gh/woocommerce/woocommerce-ios)
-[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
 
-# WooCommerce for iOS
+<h1 align="center"><img src="https://woocommerce.com/wp-content/themes/woo/images/logo-woocommerce.svg" width="300"><br>for iOS</h1>
 
-A Jetpack-powered companion app for WooCommerce.
+<p align="center">A Jetpack-powered companion app for WooCommerce.</p>
 
-## Build Instructions
+<p align="center">
+    <a href="https://circleci.com/gh/woocommerce/woocommerce-ios">
+        <img src="https://circleci.com/gh/woocommerce/woocommerce-ios.svg?style=svg" alt="CircleCI">
+    </a>
+    <a href="https://houndci.com">
+        <img src="https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg" alt="Reviewed by Hound">
+    </a>
+</p>
+
+<p align="center">
+    <a href="#-build-instructions">Build Instructions</a> • 
+    <a href="#-documentation">Documentation</a> • 
+    <a href="#-automation">Automation</a> • 
+    <a href="#-security">Security</a> •
+    <a href="#-need-help">Need Help?</a> • 
+    <a href="#-resources">Resources</a> • 
+    <a href="#-license">License</a>
+</p>
+
+## 🎉 Build Instructions
```

Comment:
nit: how about 🔨 for emoji here? just thought it matches the Xcode icon a bit 😄 totally cool to keep the tada as well!

Reply:
I tried it but it doesn't feel _inviting_. 😄 I'll keep the tada for now. I like that it signifies “Let's celebrate that you're trying to build the app!”

![image](https://user-images.githubusercontent.com/198826/95999394-16765580-0df3-11eb-9cb5-1a335e22423a.png)